generator/src/main/java/com/graphicsfuzz/generator/fuzzer/OpaqueExpressionGenerator.java

```diff
@@ -199,6 +199,16 @@ private Expr makeOpaqueZeroOrOne(boolean isZero, BasicType type, boolean constCo
           }
           return new FunctionCallExpr("exp", makeOpaqueZero(type, constContext, newDepth,
               fuzzer));
+        case 6:
+          // represent 0 as the natural logarithm of opaqueOne, e.g. log(1.0)
+          if (!isZero) {
+            continue; // log(opaqueZero) only provides a means of representing 0, not 1
```

Comment:
Should this say "log(opaqueOne) only provides ..."?

Reply:
Fixed